project.clj

```diff
@@ -68,8 +68,15 @@
   ;; perhaps because Leiningen is using this as uneval'ed code.
   ;; For now just duplicate the line.
   :test-selectors {:default (complement (fn [x]
-                                          (some->> x :ns ns-name str (re-matches #"^clara\.generative.*"))))
-                   :generative (fn [x] (some->> x :ns ns-name str (re-matches #"^clara\.generative.*")))}
+                                          (let [blacklisted-packages #{"generative" "performance"}
+                                                patterns (into []
+                                                           (comp
+                                                             (map #(str "^clara\\." % ".*"))
+                                                             (interpose "|"))
+                                                           blacklisted-packages)]
+                                            (some->> x :ns ns-name str (re-matches (re-pattern (apply str patterns)))))))
+                   :generative (fn [x] (some->> x :ns ns-name str (re-matches #"^clara\.generative.*")))
+                   :performance (fn [x] (some->> x :ns ns-name str (re-matches #"^clara\.performance.*")))}
```

Comment:
Having this live in the main Clara project seems reasonable to me, but there is an existing (old) [clara-benchmark](https://github.com/rbrush/clara-benchmark) project.  FYI @rbrush in case you have suggestions, objections, etc.

Reply:
I'm in favor of including this in the main Clara project. The test selectors are a nice solution to performance concerns here.